webpack.config.dev.js

```diff
@@ -35,18 +34,6 @@ export default {
         collapseWhitespace: true
       },
       inject: true
-    }),
-    new webpack.LoaderOptionsPlugin({
-      minimize: false,
-      debug: true,
-      noInfo: true, // set to false to see a list of every file being bundled.
```

Comment:
Do we know what these options above do? Shouldn't we also bring other over.

Reply:
I'm pretty sure the minimize option is being handled correctly. It defaults to false and I set it to true for css in our production config.

The debug is being removed from webpack, and I couldn't find much on noInfo. Maybe it's an outdated option.

Reply:
Awesome. Sounds good.